src/components/RegistrationForm/RegistrationForm.styled.js

```diff
@@ -24,60 +24,47 @@ const StyledFlexDiv = styled.div`
 
 // Styles for RegistrationForm
 export const FormWrapper = styled(StyledFlexDiv)`
-  padding: 0 16px;
+  padding: 24px 16px;
   width: 360px;
   background-color: var(--color-bg-form);
 
-  @media ${device.laptop} {
-    padding: 0 25px;
-    width: 658px;
-    border-radius: 10px;
-  }
-
-  @media ${device.desktop} {
-    width: 708px;
-    border-radius: 10px;
-  }
+  // @media ${device.laptop} {
+  //   margin: 12px 0;
+  //   padding: 0 25px;
+  //   width: 658px;
+  //   border-radius: 10px;
+  // }
+  //
+  // @media ${device.desktop} {
+  //   margin: 27px 0;
+  //   width: 708px;
+  //   border-radius: 10px;
+  // }
```

Comment:
Could be better to remove unused rules. And commented media-rules can be stay on if you want so

Reply:
This is a draft version of the code.